src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ *  in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  const arrFromSource = sourceString.split(';');
+  const newArrSplit = arrFromSource.map(item => item.split(':'));
+  const arrFilter = newArrSplit.filter(item => item.length > 1);
+
+  arrFilter.forEach(item => {
+    stylesObj[item[0].trim()] = item[1].trim();
+  });
```

Comment:
1. do not use contractions - arr, obj etc.
2. it can be chained like split().map().filter().......